queries.sql

```diff
@@ -59,3 +59,16 @@ select max(name) as name, escape_attempts from animals group by escape_attempts
 
 select avg(escape_attempts) from animals where extract(isoyear from  date_of_birth) between 1990 and 2000;
 
+ select * from animals as a join owners as o on a.owner_id = o.id where o.full_name like 'Melody Pond';
+
+select * from animals as a join owners as o on a.owner_id = o.id where a.species_id = 1;
+
```

Comment:
- [ ] Instead of adding a `WHERE` clause for the `species_id` to equal `1`, kindly join the `species` table with the rest of the tables and use the name of the species instead of its `id`. As per [this](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=List%20of%20all%20animals%20that%20are%20pokemon%20(their%20type%20is%20Pokemon).) requirement, you should use the name of the species and not its `id`.

Reply:
Ah okay Nice, Thanks lemme do it.